src/rpc/core_rpc_server_commands_defs.h

```diff
@@ -3026,29 +3008,36 @@ namespace cryptonote
       };
 
       requested_fields_t fields;
+      bool gave_if_not_equal;
```

Comment:
For `gave_if_not_equal` and `fields` you can add the comment `// @NoLokiRPCDocGen Internal use only, not serialized` and it won't display in docs when I generate the documentation.

Reply:
Added.